src/polyfills.ts

```diff
@@ -43,7 +43,7 @@
 
 /** Evergreen browsers require these. **/
 // Used for reflect-metadata in JIT. If you use AOT (and only Angular decorators), you can remove.
-import 'core-js/es7/reflect';
+// import 'core-js/es7/reflect';
```

Comment:
why remove this?

Reply:
Angular now does this under the hood so it's no longer necessary. It was giving me a compilation error.